components/tools/OmeroPy/test/integration/metadata/test_populate.py

```diff
@@ -133,10 +134,10 @@ def createDataset(self, names=("A1", "A2")):
         return ds.proxy()
 
     def createScreen(self, rowCount, colCount):
-        plate1 = self.test.importPlates(plateRows=rowCount,
-                                        plateCols=colCount)[0]
-        plate2 = self.test.importPlates(plateRows=rowCount,
-                                        plateCols=colCount)[0]
+        plate1 = self.test.import_plates(plateRows=rowCount,
```

Comment:
This should be ``plate_rows`` and ``plate_cols`` if the method from testlib is used